numpy/core/src/multiarray/compiled_base.c

```diff
@@ -1738,12 +1738,21 @@ unpack_bits(PyObject *input, int axis)
     }
 
     /* Setup output shape */
+    // Why are we not using memmove here?
```

Comment:
Added the code comment so I could add the PR comment. Is there a reason we are not using `memmove` here?